### DISCUSSION TITLE
登录出错

### DISCUSSION THREAD

**Discussion by tonylee2022** [luolongfei/freenom#226] [Q&A]
freenom增加了图形验证登录，脚本无法使用。
主人，出错了，登录 freenom 出错。错误信息：Client error: `POST https://my.freenom.com/dologin.php` resulted in a `405 Not Allowed` response:
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width (truncated...)
(Error code: 34520002)

具体是在/app/app/Console/FreeNom.php文件的第131行，抛出了一个异常。异常的内容是登录 freenom 出错。错误信息：Client error: `POST https://my.freenom.com/dologin.php` resulted in a `405 Not Allowed` response:
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width (truncated...)
(Error code: 34520002

**Comment by luolongfei** [comment]
https://github.com/luolongfei/freenom/issues/225
